pulpcore/cli/file/remote.py

```diff
@@ -29,18 +31,46 @@ def remote(ctx: click.Context, pulp_ctx: PulpContext, remote_type: str) -> None:
 
 
 lookup_options = [href_option, name_option]
+create_options = [
+    click.option("--name", required=True),
+    click.option("--url", required=True),
+    click.option("--ca-cert"),
+    click.option("--client-cert"),
+    click.option("--client-key"),
+    click.option("--connect-timeout", type=float),
+    click.option("--download-concurrency", type=int),
+    click.option("--password"),
+    click.option(
+        "--policy", type=click.Choice(["immediate", "on_demand", "streamed"], case_sensitive=False)
+    ),
+    click.option("--proxy-url"),
+    click.option("--sock-connect-timeout", type=float),
+    click.option("--sock-read-timeout", type=float),
+    click.option("--tls-validation", type=bool),
+    click.option("--total-timeout", type=float),
+    click.option("--username"),
+]
+update_options = [
+    click.option("--url"),
+    click.option("--ca-cert"),
```

Comment:
deepcopy is the kind of thing i really like to avoid.
I have been thinking about a list of fields that is then transformed into options, but that is probably at least two refactoring steps down the road.